src/app/_components/client/AvatarList/AvatarList.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
+
+import profileImage from "../../../../../public/assets/userProfile.svg";
+import Avatar from "../Avatar/Avatar";
+import { useClickAvatarList } from "./hooks";
+
+import { motion } from "framer-motion";
+import Image from "next/image";
+
+interface AvatarProps {
+  id: string;
+  avatarImage: string | null;
+  name: string;
+}
+
+export interface AvatarListProps {
+  avatars: AvatarProps[];
+}
+
+const AvatarList = ({ avatars }: AvatarListProps) => {
+  const { toggle, handleClickAvatar, handleClickBackground } = useClickAvatarList();
+
+  const variants = {
+    open: { opacity: 1, x: 0 },
+    closed: { opacity: 0, x: "-100%" },
+  };
+
+  if (avatars.length === 1) {
+    const { id, avatarImage } = avatars[0];
+    return (
+      <Avatar
+        userId={id}
+        avatarImage={avatarImage}
+      />
+    );
+  }
+
+  if (avatars.length === 0) {
+    return;
+  }
+
+  return (
+    <>
+      <div className="relative">
+        <div
+          className="flex items-center cursor-pointer"
+          onClick={handleClickAvatar}
+        >
+          {avatars.slice(0, 2).map(({ id, name, avatarImage }, index) => (
+            <div
+              key={id}
+              className={`relative w-[3rem] h-[3rem] rounded-full overflow-hidden ${index === 1 && "-ml-[2.2rem]"} ${index === 1 && "mt-[1rem]"} ${index === 0 && "z-10"}`}
```

Comment:
[ P4 ]

twJoin을 통해 조건부 처리를 해서 하나로 묶어주는건 어떨까요?

```suggestion
              className={twJoin(
                "relative w-[3rem] h-[3rem] rounded-full overflow-hidden",
                index === 1 && "ml-[2.2rem] mt-[1rem]",
                index === 0 && "z-10"
              )}
```

Reply:
넵 ~ 수정해놓겠습니다 !! 😀